src/modules/search/searchApi.js

```diff
@@ -13,8 +13,21 @@ import {
 } from '../../util/apiHelpers';
 
 const baseUrl = apiResourceUrl('/search-api/v1/search');
+const articleUrl = apiResourceUrl('/article-api/v2/articles');
```

Comment:
Kanskje greit å ha en egen `article`-mappe for `article-api`-relaterte metoder? Search-API søker også mot Article-API forsåvidt.